### PR TITLE
STIJ-1: Added Gallery molecule.

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/31-molecules/building-blocks/gallery/_gallery.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.gallery {
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/gallery/_gallery.scss]: When there are 3 thumbnails the margin/paddings aren't correct

File: components/31-molecules/building-blocks/gallery/gallery.twig [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<div class="gallery">
+  <div class="gallery-main">
+    {% include '@link' with {
+      'rel': 'gallery-1',
+      'title': 'test',
+      'modifier': 'swipebox',
+        'link': 'https://stad.gent/sites/default/files/styles/sidebar_full_width/prco/62000/images/252144-Warme%20William_beeld_Pagina_13-2dea86-original-1498657714.png?itok=wZ-Pl9vt',
+      'text': '<img src="https://stad.gent/sites/default/files/styles/sidebar_full_width/prco/62000/images/252144-Warme%20William_beeld_Pagina_13-2dea86-original-1498657714.png?itok=wZ-Pl9vt" alt="warme william" />'
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/gallery/gallery.twig]: Change it into dummy images.

File: components/_preview.twig [hunk 3]
```diff
@@ -5,14 +5,22 @@
     <meta charset="utf-8">
     <title>Preview Layout</title>
     <link href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,700,400italic" rel="stylesheet" type="text/css" />
+
     <link media="all" rel="stylesheet" href="{{ '/css/main.css' | path }}">
     <link media="all" rel="stylesheet" href="{{ '/css/styleguide.css'| path }}">
+    <link media="all" rel="stylesheet"href="https://cdnjs.cloudflare.com/ajax/libs/jquery.swipebox/1.4.4/css/swipebox.min.css">
+
```
Comment (Jeroen005) [inline components/_preview.twig]: Add readme with used jquery plugin :)

File: components/31-molecules/building-blocks/gallery/_gallery.scss (hunk 1 above)
Comment (meirege) [reply components/31-molecules/building-blocks/gallery/_gallery.scss]: Fixed margins.

File: components/_preview.twig (hunk 3 above)
Comment (meirege) [reply components/_preview.twig]: Added link to the plugin in the component readme.

File: components/31-molecules/building-blocks/gallery/gallery.twig [hunk 4]
```diff
@@ -2,36 +2,35 @@
...
     } %}
   </div>
   <div class="gallery-thumbs">
     {% include '@link' with {
       'rel': 'gallery-1',
       'modifier': 'swipebox',
-      'link': 'https://stad.gent/sites/default/files/styles/sidebar_full_width/prco/62000/images/252144-Warme%20William_beeld_Pagina_13-2dea86-original-1498657714.png?itok=wZ-Pl9vt',
-    'text': '<img src="https://stad.gent/sites/default/files/styles/sidebar_full_width/prco/62000/images/252144-Warme%20William_beeld_Pagina_13-2dea86-original-1498657714.png?itok=wZ-Pl9vt" alt="warme william" />'
+      'link': 'http://lorempixel.com/800/532/city/',
+    'text': '<img src="http://lorempixel.com/800/532/city/" alt="warme william" />'
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/gallery/gallery.twig]: https

**Pull request merged**